Extensions/xneko.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XNeko **//
-//* VERSION 1.2 REV A **//
+//* VERSION 1.2.1 REV A **//
```

Comment:
`1.2.1`. We are switching all extensions to a [semver](http://semver.org/) compliant format